app/src/main/java/com/codesoom/assignment/advice/NotFoundErrorAdvice.java

```diff
@@ -0,0 +1,25 @@
+package com.codesoom.assignment.advice;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.ErrorResponse;
+import com.codesoom.assignment.exceptions.ProductNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ControllerAdvice
+public class NotFoundErrorAdvice {
+    @ResponseBody
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    @ExceptionHandler(ProductNotFoundException.class)
+    public ErrorResponse handleProductNotFound() {
+        return new ErrorResponse("Product not Found");
+    }
+
+    @ResponseBody
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    @ExceptionHandler(UserNotFoundException.class)
+    public ErrorResponse handleUserNotFound() { return new ErrorResponse("User not found"); }
```

Comment:
이렇게 한 줄로 join하지 않고 중괄호가 있는 곳에서 개행을 해주면 좋겠습니다.
```suggestion
    public ErrorResponse handleUserNotFound() {
        return new ErrorResponse("User not found");
    }
```

Reply:
`printwidth` 가 설정이 너무 길게 되어있어서 자꾸만 개행을 해도 변경이 되었는데 이번에 옵션을 찾아서 변경했습니다!